src/unlimited.h

```diff
@@ -186,32 +200,33 @@ extern std::set<uint256> setUnVerifiedOrphanTxHash;
 extern CCriticalSection cs_xval;
 // Xpress Validation: end
 
-extern void LoadFilter(CNode *pfrom, CBloomFilter *filter);
+extern void LoadFilter(CNode* pfrom, CBloomFilter* filter);
```

Comment:
This is going from correct to incorrect for the * location IMO . Similarly for & references.